pom.xml

```diff
@@ -618,7 +618,6 @@
                     <groupId>ch.qos.logback</groupId>
                     <artifactId>logback-classic</artifactId>
                     <version>${logback.version}</version>
-                    <scope>test</scope>
```

Comment:
Why did you change this?
Is there any functionality in the driver that requires logback?

Reply:
It's in **bench** profile used for performance runs